backend/controllers/LocationsController.ts

```diff
@@ -1,47 +1,44 @@
-import { Response, Request } from 'express';
-import { TupleTypeReference } from 'typescript';
-import Location from '../models/Locations';
+import { Response, Request } from "express";
+import { TupleTypeReference } from "typescript";
+import Location from "../models/Locations";
 
 const LocationsController = {
   async getLocationsByZoom(req: Request, res: Response) {
     try {
       const center = JSON.parse(req.query.center as string);
       const bounds = JSON.parse(req.query.bounds as string);
       const name = req.query.name as string;
-      const filters = JSON.parse(req.query.filters as any)
+      const filters = JSON.parse(req.query.filters as any);
```

Comment:
maybe, instead of any, we need to create something like IFilter interface?

Reply:
done